### PR TITLE
add mentor guidelines to mentor page

### PR DESCRIPTION
**Pull request by marklocklear**
Add mentor guidelines and expectations to mentor page on the website.

## Description
Add mentor guidelines and expectations to mentor page on the website.

## Related Issue
NA

## Motivation and Context
Gives more detail on what it means to be a mentor.

## How Has This Been Tested?
Updated snapshot for mentor pages and all tests pass

## Screenshots (if appropriate):
NS

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by andrew-t-james** [comment]
@marklocklear Thanks for adding all this information. Can you add bootstrap classes to format this similarly with the rest of the document? If you are tasked out with work I can ping one of the troops who has been asking to contribute to take this task.

**Comment by marklocklear** [comment]
@andrew-t-james @ozzievee pinged me in slack and said he would add the addition of Bootstrap classes/styling.

**Comment by ozzievee** [comment]
@andrew-t-james and @marklocklear , I pushed the latest changes for review. Due to the amount of content, I took a different approach to the layout of the page. Please let me know what you think!

File: src/assets/css/main.css [hunk 1]
```diff
@@ -1529,6 +1529,10 @@ label {
   font-weight: 900;
   margin-bottom: 45px;
 }
+.contactus-brief > ul, li {
+  list-style-type: disc !important;
+  margin: 0 3rem;
```
Comment (andrew-t-james) [inline src/assets/css/main.css]: This broke the layout of the nav-bar links in the header. @ozzievee see attached image.
image

**Comment by andrew-t-james** [comment]
@ozzievee I looked this over and saw some changes made to the main style sheet are affecting the main links in the header can we adjust this back so that links are not moved? See line affecting styles here https://github.com/Vets-Who-Code/vets-who-code-app/pull/59/commits/46c6198da72f6e8643046d24a5719e02721d3fb6

**Comment by marklocklear** [comment]
Great work @ozzievee 👏👏👏 Actually I was thinking it might be nice to use the accordion style used on the syllabus page, so good call.

**Comment by ozzievee** [comment]
Adjusted the identified styles. I am unsure if I like the according title styling as it pushes the icon to the lower right rather than center right. I will see if there is a way to fix this; however, at the moment the changes should be functional and ready to ship.. of course pending review.

**Pull request merged**